README_zh.md

```diff
@@ -0,0 +1,160 @@
+[![下载次数](https://static.pepy.tech/badge/langcheck)](https://pepy.tech/project/langcheck)
+![GitHub](https://img.shields.io/github/license/citadel-ai/langcheck)
+
+这是一个简单的Python库，用于评估LLM应用程序。
+
+[安装](#安装) •
+[使用示例](#使用示例) •
+[快速开始](https://langcheck.readthedocs.io/en/latest/quickstart.html) •
+[文档](https://langcheck.readthedocs.io/en/latest/index.html) •
+[English](README.md) •
+[日本語](README_ja.md) •
+[Deutsch](README_de.md)
+
+</div>
+
+## 安装
+
+```shell
+pip install langcheck
+```
+
+如果安装不成功，请查看[常见问题解答(英文)](https://langcheck.readthedocs.io/en/latest/installation.html#installation-faq)。
+
+## 使用示例
+
+### 文本评估
+
+您可以使用各种指标来评估LLM生成的文本。
+
+```python
+from langcheck.metrics.zh import sentiment
+
+# 输入LLM生成的文本
+generated_outputs = [
+    '很高兴能够帮到您。',
+    '我很高兴回答您的问题。',
+    '我不太清楚，请自己查找。'
+]
+
+# 分析文本中的情绪表达（也可以基于阈值进行测试）
+sentiment(generated_outputs) > 0.5
+```
+
+![MetricValueWithThreshold 的屏幕截图](docs/_static/MetricValueWithThreshold_output_zh.png)
+
+通过使用`assert`，您可以轻松地将LangCheck的各种指标转换为单元测试。
+
+```python
+assert sentiment(generated_outputs) > 0.5
+```
+
+LangCheck还包含了以下用于评估LLM应用程序的指标。
+
+|                                                                 类型                                                                 |                                                   主要指标                                                    |     语言     |
+| ------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------- | ------------ |
+| [无参考文本质量指标](https://langcheck.readthedocs.io/en/latest/metrics.html#reference-free-text-quality-metrics)   | `toxicity(generated_outputs)`<br>`sentiment(generated_outputs)`                                               | 英语、日语、中文、德语 |
+| [有参考文本质量指标](https://langcheck.readthedocs.io/en/latest/metrics.html#reference-based-text-quality-metrics) | `semantic_similarity(generated_outputs, reference_outputs)`<br>`rouge2(generated_outputs, reference_outputs)` | 英语、日语、中文、德语 |
+| [基于来源的文本质量指标](https://langcheck.readthedocs.io/en/latest/metrics.html#source-based-text-quality-metrics)       | `factual_consistency(generated_outputs, sources)`                                                             | 英语、日语、中文、德语 |
+| [文本结构指标](https://langcheck.readthedocs.io/en/latest/metrics.html#text-structure-metrics)                             | `is_float(generated_outputs, min=0, max=None)`<br>`is_json_object(generated_outputs)`                         | 所有语言   |
+
+### 数值可视化
+
+LangCheck还可以使用交互式图表来可视化数值。
+
+```python
+# 选择一些指标
+sentiment_values = langcheck.metrics.zh.sentiment(generated_outputs)
+toxicity_values = langcheck.metrics.zh.toxicity(generated_outputs)
+
+# 关于一个指标的交互式散点图
+sentiment_values.scatter()
+```
+
+![Scatter plot for one metric](docs/_static/scatter_one_metric_ja.gif)
```

Comment:
@anelephantbowl Could you suggest some phrases that we can use for this demo? A few extra variations of these kinds of phrases would be helpful
```
generated_outputs = [
    '很高兴能够帮到您。',
    '我很高兴回答您的问题。',
    '我不太清楚，请自己查找。'
]
```

Also, if you can tell me what would be a reasonable word to filter for (as we do in the Japanese gif) that would be helpful!